backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -186,11 +189,17 @@ public Endpoint createEndpoint(@Context SecurityContext sec, @NotNull @Valid End
                 }
             }
 
-            if (endpointSubType.equals("slack")) {
+            if (endpointSubType != null && endpointSubType.equals("slack")) {
                 CamelProperties properties = endpoint.getProperties(CamelProperties.class);
                 String processorName = "p-" + endpoint.getAccountId() + "-" + UUID.randomUUID();
                 properties.getExtras().put(OB_PROCESSOR_NAME, processorName);
-                String processorId = setupOpenBridgeProcessor(endpoint, properties, processorName);
+                String processorId = null;
+                try {
+                    processorId = setupOpenBridgeProcessor(endpoint, properties, processorName);
+                } catch (Exception e) {
+                    LOGGER.warn("Processor setup failed: " + e.getMessage());
```

Comment:
```suggestion
                    LOGGER.warn("Processor setup failed", e);
```

Reply:
I don't think the full trace is needed, message should be enough

Reply:
I would agree if that line logged a specific exception but we're catching many different kinds of exceptions here, most of them resulting from an HTTP call failure. Also, logging the message instead of the exception itself not only discards the stacktrace but also the exception type.